lib/update/cluster/builder_test.go

```diff
@@ -489,6 +473,28 @@ func (r *params) leaderMasterPhase() storage.OperationPhase {
 				},
 				Requires: []string{t("/masters/%v/drain")},
 			},
+			{
+				ID:          t("/masters/%[1]v/elect"),
```

Comment:
nit: not sure why it was using a positional placeholder with just a single value:
```go
"/masters/%v/elect"
```